build-tools/packages/build-cli/docs/list.md

```diff
@@ -0,0 +1,41 @@
+`flub list`
+===========
+
+List packages in a release group in topological order.
```

Comment:
Question: should the toplogical sorting be optional? Only asking because it's an option with Lerna. I don't know that we have a use case for not topo-sorting.

Reply:
Good question. Right now the main goal for this command is to replace lerna in CI, so I don't want to add anything we don't need there in this initial PR.

The [flub info](https://github.com/microsoft/FluidFramework/blob/main/build-tools/packages/build-cli/docs/info.md) command is where I envision a lot of the "enumerate all FOOs in the repo" capabilities being exposed if we need them. I considered adding a `--toposort` option to the info command, but it made that command a lot more complex, partly because `info` isn't only about packages, whereas `list` is _always_ going to output a list of packages.

Long term I could imagine subcommands, like `flub info packages` ~= `flub list`. But again, no use-cases at the moment, and no plans there.